Source/XUnitExtensions/Lit/DiffCommand.cs

```diff
@@ -10,6 +10,7 @@ namespace XUnitExtensions.Lit {
   /// because 'diff' does not exist on Windows.
   /// </summary>
   public class DiffCommand : ILitCommand {
+    private static readonly bool UpdateExpectFile = false;
```

Comment:
Can you please document how to use this in the README of the tests? I'm interested but will soon forget where I had to change something.